dotnet-test-samples/sqs-lambda/src/SQSEventHandler/ProcessEmployeeFunction.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Threading.Tasks;
+using Amazon.Lambda.Core;
+using SqsEventHandler.Models;
+using SqsEventHandler.Triggers;
+
+// Assembly attribute to enable the Lambda function's JSON input to be converted into a .NET class.
+[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
+
+namespace SqsEventHandler;
+
+/// <summary>
+/// This class implements the business logic of the function. The function handler can be found in
+/// the base abstract class SqsEventTrigger::Handler
+/// </summary>
+public class ProcessEmployeeFunction : SqsEventTrigger<Employee>
```

Comment:
I love how simple this handler is, but as per the above comment I wonder if it makes it hard for someone new to Lambda/SQS to work out what is happening.

Reply:
Let me remove the adapter and have the logic just inside the trigger. This will be just one layer and we get the same simplicity at the handler.

Reply:
Removing the adapter and renaming the trigger to SqsEventHandler